src/pages/FlowRun/Actions.vue

```diff
@@ -123,6 +133,29 @@ export default {
         }
       },
       update: data => data?.task_run
+    },
+    utilityDownstreamTasks: {
```

Comment:
Can you set out why this has been moved to the Actions component?  I'm concerned that we'll be making the query every time the flow run page is loaded when it's only needed for restarts.